doc/scenario.md

```diff
@@ -0,0 +1,317 @@
+# Role Scenario
+
+The role uses a scenario of steps to determine what to do.
+
+Using a scenario of steps, you can:
+- run the specified steps in any order, but API must be followed (with `cartridge_scenario`);
+- replace the steps of the role with your own or add new steps
+  (with `cartridge_custom_steps_dir` or `cartridge_custom_steps`).
+
+## Steps
+
+In `cartridge_scenario` you should specify names of steps. The default scenario includes the following steps:
+- [deliver_package](#deliver_package)
+- [update_package](#update_package)
+- [update_instance](#update_instance)
+- [configure_instance](#configure_instance)
+- [restart_instance](#restart_instance)
+- [wait_instance_started](#wait_instance_started)
+- [connect_to_membership](#connect_to_membership)
+- [edit_topology](#edit_topology)
+- [cleanup_expelled](#cleanup_expelled)
+- [configure_auth](#configure_auth)
+- [configure_app_config](#configure_app_config)
+- [bootstrap_vshard](#bootstrap_vshard)
+- [configure_failover](#configure_failover)
+
+In addition to the steps in the default scenario, there are additional steps:
+- [set_control_instance](#set_control_instance)
+
+To replace the steps of the role with your own or add new steps, you should use `cartridge_custom_steps_dir`
+or `cartridge_custom_steps` options.
+
+## Example
+
+For example, you can run `edit_topology` step without `connect_to_membership` step (for existing cluster):
```

Comment:
## Examples

### Editing topology without connecting to membership

For example, you need to edit topology, but you don't wand to wait for a long `connect_to_membership` step (in fact this step is only needed to find some instance that is already in cluster). 

As we can see in `edit_topology` step API (add a link), it requires `control_instqnce` fact. 
Since we definitely know a name of instance that is already in cluster, we can specify it explicitly and remove `connect_to_membership` step from scenario:

```yaml
# edit_toplogy_playbook.yml

- name: My playbook
  hosts: all
  tasks:
    - name: Edit topology
      import_role:
        name: tarantool.cartridge
      vars:
        cartridge_scenario:
          - edit_topology
        control_instance:
          name: core_1
          console_sock: '/var/run/tarantool/core_1.control'
```